docs/install.rst

```diff
@@ -56,17 +56,35 @@ OS X + Docker
 Ubuntu 14.04 (manual way)
 -------------------------
 
-1. Install system dependencies (check
-`provision.sh <https://github.com/scrapinghub/splash/blob/master/dockerfiles/splash/provision.sh>`)
-
-2. Clone the repo from GitHub::
+1. Clone the repo from GitHub::
 
         $ git clone https://github.com/scrapinghub/splash/
 
-3. Install dependencies with pip::
-
-        $ cd splash
-        $ pip3 install -r requirements.txt
+2. Install dependencies::
+
+        $ cd splash/dockerfiles/splash   
+        
+        $ sudo ./provision.sh prepare_install 
+        
+        $ sudo ./provision.sh install_msfonts 
+        
+        $ sudo ./provision.sh install_builddeps
+        
+        $ sudo ./provision.sh install_deps
+        
+        $ sudo ./provision.sh install_extra_fonts
+       
+        $ sudo ./provision.sh install_pyqt5
+       
+        $ sudo ./provision.sh install_python_deps
+       
+        $ sudo ./provision.sh remove_builddeps
+       
+        $ sudo ./provision.sh remove_extra
```

Comment:
These steps are really aggressive - they are to trim Docker image size. For example, remove_extra removes all man files (documentation) from the system. I think they shouldn't be used on user machines, this is dangerous.